scripts/configurator/parser/cmake_var.py

```diff
@@ -1,9 +1,108 @@
-from typing import Dict, Any, Type, Mapping
+from typing import Dict, Any, Optional, Type, Mapping, List
+import re
+from sys import stderr
+
+
+class metadata:
+   keyword_expr = re.compile(r"([A-Z_]+)")
+
+   def __init__(self, comment: List[str]) -> None:
+      self.depends_on: Optional[List[str]] = None
+      self.group: Optional[str] = None
+      self.realtype: Optional[str] = None
+
+      self.kw_to_parse: Mapping[str, function] = {
```

Comment:
3 dictionaries with the same key type are too many and that's a sign that something is wrong. In this case we should have just _one_ dictionary KEYWORD -> value with value being a tuple (e.g. namedTuple) containing: getter, setter, parser. This will make this whole code way shorter.

Reply:
changed